pyaerocom/aeroval/modelmaps_helpers.py

```diff
@@ -5,6 +5,7 @@
 from cartopy.mpl.geoaxes import GeoAxes
 from matplotlib.axes import Axes
 from matplotlib.colors import ListedColormap, to_hex
+from numpy import round
```

Comment:
Slightly confusing import since `np.round` overwrites here the built-in `round`.

Reply:
cleaner: 
```python3
import numpy as np
...
np.round(...)
```